packages/brian/example/package.json

```diff
@@ -15,21 +15,21 @@
     "@freesewing/mui-theme": "latest",
     "@freesewing/pattern-info": "latest",
     "@freesewing/plugin-bundle": "latest",
-    "@freesewing/plugin-theme": "latest",
     "@freesewing/plugin-i18n": "latest",
     "@freesewing/plugin-svgattr": "latest",
+    "@freesewing/plugin-theme": "latest",
```

Comment:
I'm sorry to say that this file is generated via `yarn reconfigure`. So any manual changes will eventually be overwritten. To make changes to dependencies you have to update [config/dependencies.yaml](https://github.com/freesewing/freesewing/blob/develop/config/dependencies.yaml), then run `yarn reconfigure`.

Side note: I think we should add a PR build step that checks whether yarn reconfigure changes anything. It would also be nice to have a comment in this file but I'm not sure that is possible.